unity-client/Assets/Scripts/MainScripts/DCL/Components/Textures/DCLTexture.cs

```diff
@@ -1,8 +1,8 @@
 using DCL.Components;
 using DCL.Controllers;
-using DCL.Helpers;
 using System;
 using System.Collections;
+using Cinemachine;
```

Comment:
Is it needed to import `Cinemachine` here?

Reply:
Nope, fixed it :)